blog/content/post/2023-12-14-aem_phase3.md

```diff
@@ -0,0 +1,318 @@
+---
+title: TrenchBoot Anti Evil Maid - Phase 3
+abstract: "This blog post marks completion of next phase of TrenchBoot Anti Evil
+           Maid project for Qubes OS. Even though user experience didn't change
+           too much, the implementation went through a major overhaul."
+cover: /covers/trenchboot-logo.png
+author: krystian.hebel
+layout: post
+published: true
+date: 2023-12-14
+archives: "2023"
+
+tags:
+  - QubesOS
+  - TrenchBoot
+  - GRUB2
+  - Xen
+  - Hypervisor
+categories:
+  - Firmware
+  - Bootloader
+  - Hypervisor
+  - OS Dev
+  - Security
+
+---
+
+## Introduction
+
+If you haven't read the previous [TrenchBoot AEM blog
+post](https://blog.3mdeb.com/2023/2023-01-31-trenchboot-aem-for-qubesos/) or
+our [project plan](https://docs.dasharo.com/projects/trenchboot-aem-v2/) yet,
+it may be a good lecture to catch up on what the TrenchBoot AEM project is
+about.
+
+## What's new in TrenchBoot AEM?
+
+This phase doesn't really add any new features visible from user's point of
+view. Instead, it focused mostly on bringing the code up to current Secure
+Launch implementation, which was still evolving when we started the AEM project.
+With Secure Launch code brought up to speed, we should be able to put more
+effort towards upstreaming soon.
+
+Taught by our hiccups with earlier phases, we also changed our approach to
+testing. Now we don't check only if the PCRs were extended, but also whether
+their values are correct. Qubes OS packages, created by CI, are now used
+wherever possible, and manually built Xen and/or GRUB are used only for
+development and debugging outside of Qubes OS.
+
+### Secure Launch specification
+
+Secure Launch specification was created as an attempt to provide a unified
+experience in passing configuration and other meta-data to a TrenchBoot Secure
+Launch entry point across all supported architectures. Latest version of
+specification can be found on [TrenchBoot
```

Comment:
Nit: I think `the` is missing:
```suggestion
the specification can be found on [TrenchBoot
```
And maybe the URL should be https://trenchboot.org/specifications/Secure_Launch/?  That's what I expected to see.

Reply:
https://github.com/3mdeb/news-and-ideas/pull/561/commits/d87984353d98b14ec17191cbad5b122f2ec048fa